openassessment/templates/openassessmentblock/oa_team_uploaded_files.html

```diff
@@ -10,7 +11,9 @@ <h5 class="submission__answer__part__text__title">
         {% endif %}
 
         <div class="submission__answer__files">
-        {% for file_url, file_description, file_name, owner_username in team_file_urls %}
+        {% for file in team_file_urls %}
+        {% with file_url=file|get_item:"download_url" file_description=file|get_item:"description" file_name=file|get_item:"name" owner_username=file|get_item:"uploaded_by" %}
```

Comment:
This is hard to read... is there any way to break it up or put this query into js?

Reply:
Not that I'm aware of... I can break it up more so its not all on one line and just use the filters where the variable is used, but this makes the change-set easier to grok. It'd basically be the same thing but spread out.

Reply:
I stand corrected. @jansenk's syntax works and removes the need for a filter.